DEC/src/roe/SSAExtract.jl

```diff
@@ -0,0 +1,137 @@
+module SSAExtract
```

Comment:
Maybe just call this SSAs. And then all the things prefixed with SSA (i.e. SSAVar, etc.) can be `SSAs.Var`, etc.